tools/fab/frontend-install.py

```diff
@@ -1,16 +1,35 @@
-#!/usr/bin/python
+#! /opt/stack/bin/python3
```

Comment:
What happens to when this script is run as `python2 frontend-install.py`?  Does it start to run and then traceback?  If so, can we put a check early on to actually verify that this is running against a python major version == 3 and print an appropriate error message?

I don't normally think this kind of thing is required, but since people are used to fetching this script directly from our tree, it might help avoid some questions.